src/search.js

```diff
@@ -114,14 +122,14 @@ const matches = (terms) => {
 // :thumbs up: => ['thumbs', 'up']
 const parse = query => query.replace(/[:]/g, '').split(/\s+/)
 
-module.exports = function search (query, skinTone, pasteByDefault = false) {
-  if (pasteByDefault) resetWordsForPasteByDefault()
-
+function search (query, skinTone, operationType = 'clipboard') {
   setSkinToneModifier(skinTone)
 
-  if (!query) return all()
+  if (!query) return all(operationType)
 
   const terms = parse(query)
 
-  return alfredItems(matches(terms))
+  return alfredItems(matches(terms), operationType)
 }
+
+module.exports = { search, all, alfredItems, alfredItem, getSubtitleStrings }
```

Comment:
The purpose of this file is to provide the `search` function. It should directly export that function, i.e. `module.exports = function search...`.

In order to test internal methods, we would want to export them as internals by:

```js
module.exports.internals = {
  all,
  alfredItems,
  // etc
}
```